phoenix-core-client/src/main/java/org/apache/phoenix/filter/SkipScanFilter.java

```diff
@@ -91,27 +92,39 @@ private enum Terminate {AT, AFTER};
     public SkipScanFilter() {
     }
 
-    public SkipScanFilter(SkipScanFilter filter, boolean includeMultipleVersions) {
-        this(filter.slots, filter.slotSpan, filter.schema, includeMultipleVersions);
+    public SkipScanFilter(SkipScanFilter filter, boolean includeMultipleVersions,
+            boolean isMultiKeyPointLookup) {
+        this(filter.slots, filter.slotSpan, filter.schema, includeMultipleVersions,
+                isMultiKeyPointLookup);
     }
 
-    public SkipScanFilter(List<List<KeyRange>> slots, RowKeySchema schema) {
-        this(slots, ScanUtil.getDefaultSlotSpans(slots.size()), schema);
+    public SkipScanFilter(List<List<KeyRange>> slots, RowKeySchema schema, boolean isMultiKeyPointLookup) {
+        this(slots, ScanUtil.getDefaultSlotSpans(slots.size()), schema, isMultiKeyPointLookup);
     }
 
-    public SkipScanFilter(List<List<KeyRange>> slots, int[] slotSpan, RowKeySchema schema) {
-        this(slots, slotSpan, schema, false);
+    public SkipScanFilter(List<List<KeyRange>> slots, int[] slotSpan, RowKeySchema schema,
+            boolean isMultiKeyPointLookup) {
+        this(slots, slotSpan, schema, false, isMultiKeyPointLookup);
     }
     
-    private SkipScanFilter(List<List<KeyRange>> slots, int[] slotSpan, RowKeySchema schema, boolean includeMultipleVersions) {
-        init(slots, slotSpan, schema, includeMultipleVersions);
+    private SkipScanFilter(List<List<KeyRange>> slots, int[] slotSpan, RowKeySchema schema,
+            boolean includeMultipleVersions, boolean isMultiKeyPointLookup) {
+        init(slots, slotSpan, schema, includeMultipleVersions, isMultiKeyPointLookup);
     }
     
     public void setOffset(int offset) {
         this.offset = offset;
     }
+    public boolean isMultiKeyPointLookup() {
+        return isMultiKeyPointLookup;
+    }
+
+    public List<KeyRange> getPointLookupKeyRanges() {
+        return slots.get(0);
```

Comment:
This API should check if isMultiKeyPointLookup is true then return the 0th element else return an empty list. This will be similar to how some of the APIs in ScanRanges class work like `getPointLookupCount` and `getPointLookupKeyIterator`